src/manage_nsfs/nc_utils.js

```diff
@@ -0,0 +1,44 @@
+/* Copyright (C) 2024 NooBaa */
+'use strict';
+
+const dbg = require('../util/debug_module')(__filename);
+const path = require('path');
+const P = require('../util/promise');
+const nb_native = require('../util/nb_native');
+const native_fs_utils = require('../util/native_fs_utils');
+
+/**
+ * get_account_by_id will return the account
+ * TODO: when we have the config files by id, we can account config by id
+ * @param {object} global_config
+ * @param {string} account_id
+ */
+async function get_account_by_id(global_config, account_id) {
```

Comment:
I think this should be an abstract method of Bucketspace, and implement it differently for bucketspace_fs/nb.

Reply:
This is an interesting idea, but we need to think about it more, for example:
1) Will it still be relevant after we have the new config structure?
2) How do we want to improve out code and write in one-place functions that will serve: namespace_fs, bucketspace_fs, accountspace_js and the manage_nsfs?

Reply:
1. yes.
2. so actually I mixed up and I think it should be an abstract method of the AccountSpace from responsibility perspective. we should be able to pass the accountspace instance to bucketspace/namespace as needed if they need to call it. for manage_nsfs it should be able to use accountspace as well to share the logic.

Reply:
Thanks for the explanation, the implementation of this would be in a separate PR.